src/ProjectNeon/Assets/Battle/Enemies/EnemyVisualizer.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class EnemyVisualizer : MonoBehaviour
+{
+
+    [SerializeField] private EnemyArea enemyArea;
+    [SerializeField] private List<GameObject> enemies;
+
+    /**
+     * @todo #29:15min Draw enemy characters on battle screen. Enemy positioning must be relative to 
+     *  total enemy count. Each enemy must dinamically create a sprite inside EnemyArea Object.
+     */
+    /**
+     * @todo #29:30min Wire enemy setup with correct events. Enemy setup on battle scene must be triggered

```

Comment:
Wiring up a single event only takes 5 minutes.

I can create a tutorial if this is unfamiliar.